addons/mail/tests/test_user_modify_own_profile.py

```diff
@@ -11,6 +11,11 @@ class TestUserModifyOwnProfile(HttpCaseWithUserDemo):
     def test_user_modify_own_profile(self):
         """" A user should be able to modify their own profile.
         Even if that user does not have access rights to write on the res.users model. """
-
+        if 'hr.employee' in self.env and not self.user_demo.employee_id:
+            self.env['hr.employee'].create({
+                'name': 'Marc Demo',
+                'user_id': self.user_demo.id,
+            })
+        self.user_demo.tz = "Europe/Brussels"
```

Comment:
Hello, this is not ideal, what makes creating an employee in this test necessary if hr is installed?

Reply:
Thanks for the hint! Sounds good, better merge this PR faster and handle that separately if the clean is not obvious.